src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculator.kt

```diff
@@ -77,13 +78,14 @@ class ChangeLevelCalculator(
       .also { log.debug("Ogrs Points: $it") }
 
   private fun getIomNominalPoints(registrations: Collection<Registration>): Int =
-    // We don't care about the full list, only if there is IOM Nominal
     when {
       registrations.any { ComplexityFactor.from(it.type.code) == ComplexityFactor.IOM_NOMINAL } -> 2
       else -> 0
     }.also { log.debug("IOM Nominal Points: $it") }
 
   companion object {
     private val log = LoggerFactory.getLogger(this::class.java)
+    private val noMandateTier = TierLevel(ZERO, 0, mapOf(NO_MANDATE_FOR_CHANGE to 0))
```

Comment:
If these are essentially constants now are they not `NO_MANDATE_TIER` etc I'm not sure what the Kotlin style is.